apiserver/common/networkingcommon/types.go

```diff
@@ -479,10 +479,9 @@ func interfaceAddressToNetworkConfig(interfaceName, configType string, address n
 			ipNet = &net.IPNet{IP: ip}
 		}
 	}
-	if ip.To4() == nil {
-		logger.Debugf("skipping observed IPv6 address %q on %q: not fully supported yet", ip, interfaceName)
-		// TODO(dimitern): Treat IPv6 addresses as empty until we can handle
-		// them reliably.
+	if ip.To4() == nil && ip.IsLinkLocalUnicast() {
```

Comment:
What do we do with the other IPv6 addresses?
Would it be more obvious if we did:

if ip.To6() != nil && ip.IsLinkLocalUnicast() ?

I almost wonder if for now we'd want to break this up into:

if ip.IP6() != nil {
  if ip.IsLinkLocalUnicast() {
    logger.Debugf("skipping observed IPv6 link-local...") 
  } else {
   logger.Debugf("including ipv6 address %s")
  }
}